src/components/App.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component} from 'react';
```

Comment:
Do you have prettier setup correctly?

It should solve for minor issues like proper spacing.

Reply:
No but we've set it up